1904-11-14.xml

```diff
@@ -0,0 +1,2181 @@
+                </div>
+            </div>
+            <div type="item">
+                <head>CAIRO TRAMS.</head>
+                <p>More than once it has been a pleasure to us to acknowledge the efficiency of the
+                    services of the Cairo Tramway Company and the evident desire of the management
+                    to meet the wishes of the public in every possible manner. We have therefore no
+                    hesitation in calling the attention of the manager to the irregularity of the
+                    service between the Citadel and the Esbekieh that has for some time past been
+                    very noticeable. A correspondent who uses this service daily com- plains that of
+                    late, sometimes on two or three mornings in succession, he has had to wait ten
+                    minutes or more for a car to find, when one did arrive, that it was not only
+                    packed full but was garlanded by a picturesque array of passengers clinging to
+                    every available stanchion and handle. Such incidents are not perhaps wholly
+                    avoidable. Accidents will happen, as the old adage has it, in the best regulated
+                    families, but weare afraid that in many instances the cause of these. delays is
+                    to be attributed to the want of tact on the part of the police, who when called
+                    upon to deal with a refractory passenger, or to act upon the complaint of some
+                    one, proceed to treat the affair in a leisurely manner that is not at all
+                    justifiable. If, as occurs from time to time, the police find it necessary to
+                    make a more or less extended enquiry, or to require the attendance of a
+                    passenger, or one of the company's officials serving with the car, it should be
+                    sufficient for them to detain the persons necessary and permit the car to
+                    proceed. If it be necessary, in the in<cb/><cb n="4"/>terests of justice or
+                    otherwise, to detain the driver or conductor of a car, we see no reason why this
+                    should not be done and the place of the man so delayed taken by the
+                    corresponding official of the next car to arrive at the spot and making the same
+                    journey. In this way all needs would be filled and the sight now to too often
+                    seen of a string of four or more cars blocking up the traffic, while a policeman
+                    makes a close inquest into some trivial complaint, would not be so common. We
+                    cannot but think that if the company would make a suggestion of this kind to the
+                    police they would be met in every possible way and the public would profit
+                    greatly.</p></div>
+           <div type="item">
```

Comment:
this div needs formatting. It's a bit involved. Treat cables as individual divs (type="cable"), each with its own <dateline> and <title>. This section is sometimes spread over two pages; when this is the case, you can use the next and prev attributes. You might wish to look at some complete issues in the repository to see the format.